### PR TITLE
Document release process

### PR DESCRIPTION
**Pull request by stefannica**

### CODE REVIEW

File: Release.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+#### Release Artifacts
+
+* example codesets
+* example workflows
+* example webapps
+
+#### Release changes
+
+* update the workflow examples to point to the released container image tags. Example: https://github.com/fuseml/examples/pull/11
+* build/release container images for webapp examples
```
Comment (flaviodsr) [inline Release.md]: I wonder if its really needed to build a new image for the webapp. AFAIK it does not include anything related to fuseml and the only resource that references it is the knative service that is also part of the repository.

**Comment by jsuchome** [comment]
Nice. Just one minor request, could you make the subtitles actually links to relevant github repos?

File: Release.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+### fuseml/extensions
+
+#### Release artifacts
+
+* extension container images
+  * mlflow-builder
+  * kfserving-predictor
+* mlflow tracker container image
+* extension installer descriptor files
```
Comment (jsuchome) [inline Release.md:58]: Please mention https://github.com/fuseml/extensions/pull/28 as an example

File: Release.md (hunk 1 above)
Comment (stefannica) [reply Release.md]: Okay, I'll mark it as optional.

File: Release.md (hunk 2 above)
Comment (stefannica) [reply Release.md:58]: Done

**Review by jsuchome — APPROVED** [review]

**Review by jsuchome — APPROVED** [review]

**Pull request merged**